<?php

function inverso($x) {
    if (!$x) {
        throw new Exception('DivisiÃ³n por cero.');
    }
    return 1/$x;
}


try {
    echo inverso(0) . "\n";
} catch (Exception $e) {
    echo 'ExcepciÃ³n capturada: ',  $e->getMessage(), "\n";
}
